cmd/pipeline/main.go

```diff
@@ -849,10 +850,10 @@ func main() {
 							"eks": clusteradapter.NewEKSService(eks.NewService(
 								clusterStore,
 								eksadapter.NewClusterManager(workflowClient, config.Pipeline.Enterprise),
-								eksadapter.NewNodePoolStore(db),
+								awscommonadapter.NewNodePoolStore(db),
```

Comment:
Why is this being moved to awscommon? EKS and PKE use entirely separate database tables.